core/app/models/spree/payment.rb

```diff
@@ -127,19 +127,26 @@ def payment_source
       res || payment_method
     end
 
-    # @return [Boolean] true when this payment is risky based on address
+    # @return [Boolean] true when this payment is risky
+    def risky?
```

Comment:
Where are we using this new method? Unless I'm missing something, we're now using [the scope](https://github.com/solidusio/solidus/blob/1c4b1ffbcf08ff6d907bdb2c77699b67a331b4fe/core/app/models/spree/payment.rb#L57), which doesn't go through it. Is is there so that payment implementations can use it? 🤔

Reply:
After talking with @kennyadsl I went for not deprecating the avs/cvv risky methods and using `#risky?` in the `_gateway` partial to give a notice for risky payments and in the payment list.